### PR TITLE
Got started on testing / added modular css

### PR DESCRIPTION
**Pull request by Windsvept**

### CODE REVIEW

File: client/src/index.jsx [hunk 1]
```diff
@@ -1,158 +1,153 @@
...
-		</div>)
-	}
-}
-
-ReactDOM.render(
-  <Header />,
-  document.getElementById('app')
-);
\ No newline at end of file
+  render() {
```
Comment (Megaloden106) [inline client/src/index.jsx]: pretty big refactoring haha

File: package.json [hunk 2]
```diff
@@ -1,5 +1,34 @@
 {
   "name": "",
+  "jest": {
```
Comment (Megaloden106) [inline package.json]: Were you able to get jest working?

Comment (Megaloden106) [reply package.json]: I did something similar, but moved to its on config file

File: tests/sum.test.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+const sum = require('./sum');
```
Comment (Megaloden106) [inline tests/sum.test.js]: Copied my test! :)

File: client/src/index.jsx [hunk 4]
```diff
@@ -1,158 +1,153 @@
...
+          <button>Share</button>
+          <button>Save</button>
+        </div>
+      </div>
+      <h1>Testing</h1>
+      <form>
+        <input type="text" className={styles.input} onChange={(e)=> { this.setState({input: e.target.value})}} />
+        <input type="submit" onClick={this.changeRestaurant.bind(this)} />
+      </form>
+      <div className={styles.tempMap}>Map Data
```
Comment (nick567187) [inline client/src/index.jsx]: good work on all the modular css. can't wait to see how the front end looks now

File: package.json [hunk 5]
```diff
@@ -1,5 +1,34 @@
...
+    "verbose": true,
```
Comment (nick567187) [inline package.json]: Also having problem with jest, is this how you got it to work?

**Pull request merged**